docker/Dockerfile.jupyter

```diff
@@ -7,11 +7,9 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && pip3 install \
 	graphviz \
-	jupyterlab \
- && ipython profile create \ 
- && echo "c.InteractiveShellApp.exec_lines = [ \
-	'import sys; sys.path.append(\"/simbricks/experiments\")' ]" \
-	>>`ipython locate`/profile_default/ipython_config.py
+	jupyterlab
+# Add simbricks experiment modules to Python path.
+ENV PYTHONPATH=/simbricks/experiments
```

Comment:
Yes, it does.

Reply:
Actually I'm wondering if this should not just be set in `docker/Dockerfile` and  `docker/Dockerfile.min`... As this is not really a jupyter issue. Want to put that into a PR on the simbricks repo?

Reply:
Makes sense. Maybe I can go even further and rename run.py to a proper executable name and add it to PATH. This way, it can be used  to run simbricks experiments from any directory and abstracts away simbrick's installation directory.